src/br/com/involves/brawlhalla/service/LegendService.java

```diff
@@ -0,0 +1,10 @@
+package br.com.involves.brawlhalla.service;
+
+import armas.Weapon;
+import principal.Legend;
+
+public interface LegendService {
+
+	Legend save(String nome, Integer precisao, Integer destreza, Integer forca, Integer defesa, Weapon one, Weapon two);
```

Comment:
Os nomes dos parâmetros das armas não são claros quanto a o que representam. Além do que estão em inglês, diferente dos demais parâmetros. Os nomes dos parâmetros referente às armas poderiam ser mudados para o português (Ex: `arma1` e `arma2`), ou todos para inglês (Ex: `name`, `accuracy`, `dexterity`, `strength`, `defense`, `weapon1` e `weapon2`).

```suggestion
	Legend save(String nome, Integer precisao, Integer destreza, Integer forca, Integer defesa, Weapon arma1, Weapon arma2);
```

```suggestion
	Legend save(String name, Integer accuracy, Integer dexterity, Integer strength, Integer defense, Weapon weapon1, Weapon weapon2);
```